superset/dataframe.py

```diff
@@ -183,7 +205,7 @@ def agg_func(cls, dtype, column_name):
         if (
             hasattr(dtype, "type")
             and issubclass(dtype.type, np.generic)
-            and np.issubdtype(dtype, np.number)
+            and dtype._is_numeric
```

Comment:
Actually, this method is Pandas specific. I'll have to use this and the previous one together.